plugins/nodes/src/js/data/MesosMastersLeader.js

```diff
@@ -0,0 +1,53 @@
+import { Observable } from "rxjs/Observable";
+import "rxjs/add/observable/timer";
+import "rxjs/add/operator/do";
+import "rxjs/add/operator/map";
+import "rxjs/add/operator/retry";
+import "rxjs/add/operator/filter";
+
+import MesosStateStore from "#SRC/js/stores/MesosStateStore";
+import { findNestedPropertyInObject } from "#SRC/js/utils/Util";
+
+const STORE_POLL_INTERVAL = 2000;
+
+function isEmptyObject(obj) {
+  return Object.keys(obj).length === 0;
```

Comment:
we have `ValidatorUtil.isEmpty` would it do the trick in your case?

Reply:
I think that would be way too much logic for what I need. I would like to have an isEmptyObject. But I don't feel like using the isEmpty on Validator